### PR TITLE
test: strict model update with mongo operators

### PR DESCRIPTION
**Pull request by b-admike**
### Description

Regression test for https://github.com/strongloop/loopback-datasource-juggler/issues/1723. It should be passing with changes in https://github.com/strongloop/loopback-datasource-juggler/pull/1726.

### Checklist

- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by b-admike** [comment]
@strongloop/sq-lb-apex @mitsos1os PTAL

**Comment by mitsos1os** [comment]
@b-admike LGTM :+1:

**Comment by dhmlau** [comment]
@slnode test please

File: test/mongodb.test.js [hunk 1]
```diff
@@ -1522,6 +1522,28 @@ describe('mongodb connector', function() {
         });
       });
     });
+
+    it('should allow extended operators in update data for strict models', function() {
+      let noteModel = db.define('noteModel', {
```
Comment (3z3qu13l) [inline test/mongodb.test.js]: const instead of let ?

Comment (b-admike) [reply test/mongodb.test.js]: Thanks @3z3qu13l, fixed.

**Pull request merged**